packages/react-textarea/src/components/TextArea/useTextAreaStyles.ts

```diff
@@ -0,0 +1,246 @@
+  //     // (This could be done without trimming using `background: linear-gradient(...)`, but using
+  //     // borderBottom makes it easier for people to override the color if needed.)
+  //     ...shorthands.borderBottom(tokens.strokeWidthThick, 'solid', tokens.colorCompoundBrandStroke),
+  //     clipPath: `inset(calc(100% - ${tokens.strokeWidthThick}) 0 0 0)`,
+
+  //     // Animation for focus OUT
+  //     transform: 'scaleX(0)',
+  //     transitionProperty: 'transform',
+  //     transitionDuration: motionDurations.ultraFast,
+  //     transitionDelay: motionCurves.accelerateMid,
+  //   },
+  //   ':focus-within:after': {
+  //     // Animation for focus IN
+  //     transform: 'scaleX(1)',
+  //     transitionProperty: 'transform',
+  //     transitionDuration: motionDurations.normal,
+  //     transitionDelay: motionCurves.decelerateMid,
+  //   },
+  //   ':focus-within:active:after': {
+  //     // This is if the user clicks the field again while it's already focused
+  //     borderBottomColor: tokens.colorCompoundBrandStrokePressed,
+  //   },
+  //   ':focus-within': {
+  //     outlineWidth: tokens.strokeWidthThick,
+  //     outlineStyle: 'solid',
+  //     outlineColor: 'transparent',
+  //   },
+  // },
+
+  filledDarker: {
+    backgroundColor: tokens.colorNeutralBackground3,
+    ...shorthands.border(tokens.strokeWidthThin, 'solid', tokens.colorTransparentStrokeInteractive),
+  },
+
+  filledLighter: {
+    backgroundColor: tokens.colorNeutralBackground1,
+    ...shorthands.border(tokens.strokeWidthThin, 'solid', tokens.colorTransparentStrokeInteractive),
+  },
+
+  outline: {
+    backgroundColor: tokens.colorNeutralBackground1,
+    ...shorthands.border(tokens.strokeWidthThin, 'solid', tokens.colorNeutralStroke1),
+    borderBottomColor: tokens.colorNeutralStrokeAccessible,
+  },
+  outlineInteractive: {
+    ':hover': {
+      ...shorthands.border(tokens.strokeWidthThin, 'solid', tokens.colorNeutralStroke1Hover),
+      borderBottomColor: tokens.colorNeutralStrokeAccessibleHover,
+    },
+
+    ':active': {
+      ...shorthands.border(tokens.strokeWidthThin, 'solid', tokens.colorNeutralStroke1Pressed),
+      borderBottomColor: tokens.colorNeutralStrokeAccessiblePressed,
+    },
+
+    ':focus-within': {
+      ...shorthands.border(tokens.strokeWidthThin, 'solid', tokens.colorNeutralStroke1),
+      borderBottomColor: tokens.colorCompoundBrandStroke,
+    },
+  },
+});
+
+/**
+ * Styles for the textArea slot
+ */
+const useTextAreaStyles = makeStyles({
+  base: {
+    ...shorthands.borderStyle('none'),
+    ...shorthands.margin('0'),
+    backgroundColor: 'transparent',
+    color: tokens.colorNeutralForeground1,
+
+    '::placeholder': {
+      color: tokens.colorNeutralForeground4,
+      opacity: 1,
+    },
+
+    '::selection': {
+      color: tokens.colorNeutralForegroundInverted,
+      backgroundColor: tokens.colorNeutralBackgroundInverted,
+    },
+
+    ':focus-visible': {
```

Comment:
This looks like it should be handled by the focus indicator styles, and removed from here. Removing outline focus styles separately from adding in other focus styles increases the chance of bugs where the focus doesn't show up at all, especially if we change or tweak the implementation later.